src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+import * as fse from "fs-extra";
```

Comment:
Should show recently used file as "recently used" at the top

Reply:
Updated here:
https://github.com/Azure/bicep/commit/25fc09c8d76f44ea4123b6c9f6defcc88b2bf23c

Reply:
Fixed